src/main/resources/db/migration/V1_0__Create_User.sql

```diff
@@ -0,0 +1,11 @@
+--------------
+---- User ----
+--------------
+
+create table if not exists users (
+    id          serial constraint user_pk primary key,
```

Comment:
not a serial, but an uuid. there already is a specific postgres module that you can activate and you'll have the uuid type. i'll leave you with the research for that